src/edu/wright/cs/raiderplanner/model/Task.java

```diff
@@ -72,6 +75,24 @@ public boolean isCheckedComplete() {
 		return canCheckComplete() && checkedComplete;
 	}
 
+	/**
+	 * Used for deadline notifications.
```

Comment:
The comment for this method and the next should be better thought out. Also, please don't use dummy content like the `.` in `@return.` just to get CheckStyle to be quiet. Think through what another programmer using this method from outside the class with no context as to how it is implemented would need/want to know to use it effectively.